Sources/XLPagerTabStrip/ButtonBarPagerTabStripViewController.swift

```diff
@@ -105,38 +105,42 @@ open class ButtonBarPagerTabStripViewController: PagerTabStripViewController, Pa
 
     open override func viewDidLoad() {
         super.viewDidLoad()
-        
+
+        #if SWIFT_PACKAGE
```

Comment:
You could have done an extension on Bundle with this if/else
But anyway the task is to drop cocoa pods...